tests/periph_rtc/main.c

```diff
@@ -77,30 +79,40 @@ int main(void)
     /* set RTC */
     print_time("  Setting clock to ", &time);
     rtc_set_time(&time);
+    start_usecs = xtimer_now_usec();
 
     /* read RTC to confirm value */
-    rtc_get_time(&time);
-    print_time("Clock value is now ", &time);
+    struct tm time_now;
+    rtc_get_time(&time_now);
+    print_time("Clock value is now ", &time_now);
 
     /* set initial alarm */
     inc_secs(&time, PERIOD);
     print_time("  Setting alarm to ", &time);
     rtc_set_alarm(&time, cb, &rtc_mtx);
+    alarm_expected_usecs = start_usecs + US_PER_SEC * PERIOD;
 
     /* verify alarm */
     rtc_get_alarm(&time);
     print_time("   Alarm is set to ", &time);
     puts("");
 
+    printf("[%" PRIu32 "] First alarm expected at %" PRIu32 " microseconds\n",
+    xtimer_now_usec(),
+    alarm_expected_usecs);
     while (1) {
         mutex_lock(&rtc_mtx);
-        puts("Alarm!");
-
+        uint32_t now_usecs = xtimer_now_usec();
+        printf("[%" PRIu32 "] Alarm! after %" PRIu32 " microseconds (error %" PRId32 " microseconds)\n",
```

Comment:
In order not to exceed the line length of 100 characters and for better indents:
```suggestion
        printf("[%" PRIu32 "] Alarm! after %" PRIu32 " microseconds "
               "(error %" PRId32 " microseconds)\n",
               now_usecs, now_usecs - start_usecs,
               (int32_t)now_usecs - alarm_expected_usecs);```

Reply:
Done, thanks!

Reply:
It might be a bit nitpicking, please fiddle the indents of the parameters a a bit so that they are all aligned with the format string.